static/js/collapsible-filters/interactions.js

```diff
@@ -123,6 +123,18 @@ export default class Interactions {
       eventType: 'update',
       storageKey: 'query',
       callback: () => {
+        const repeatedSearchTriggers = [
+          'filter-component',
+          'pagination',
+          'suggest',
+          'initialize',
+          'query-parameter'
+        ];
+        const queryTrigger = ANSWERS.core.storage.get('queryTrigger');
+        const isRepeatedSearch =  repeatedSearchTriggers.includes(queryTrigger);
```

Comment:
I don't suppose we could go the reverse direction? If the `queryTrigger` is equal to a particular value, then we set `pendingQueryUpdate`. That may make the code a bit simpler. But, I recognize that trigger may not exist.

Reply:
This is actually every single query trigger. We could have it be that if there is no `queryTrigger`, `set pendingQueryUpdate` I had this originally but felt like it was a little less obvious what's happening? On the off chance we add another trigger

Reply:
Actually I can put up a PR to add a trigger for searchbar searches? I was worried about breaking somebody's random code, but I think I'd rather do that than add a trap for us in the future